Scripts/edit_user_info.php

```diff
@@ -12,32 +12,62 @@
 $emailUpdated = false;
 
 if (strlen($firstName) > 0) {
+    // Get old first name for records
+    $logsql = "SELECT `FirstName` FROM `PortfolioTracker`.`Users` WHERE `Users`.`ID`='$userID';";
+    $oldFirstName = $conn->query($logsql); 
+    $oldFirstName = $conn->fetch_row(); // Gives old first name
```

Comment:
Change on line 18.  instead of $conn->fetch_row() should be $oldFirstName->fetch_row()